docs/src/explanations/width-inference.md

```diff
@@ -16,22 +16,22 @@ The width of a conditionally valid expression is the width of its input expressi
 
 Hardware operators have output widths as defined by the following set of rules:
 
-| operation        | bit width           |
-| ---------        | ---------           |
-| `z = x + y` *or* `z = x +% y`        | `w(z) = max(w(x), w(y))`   |
-| `z = x +& y`       | `w(z) = max(w(x), w(y)) + 1`   |
-| `z = x - y` *or* `z = x -% y`       | `w(z) = max(w(x), w(y))`    |
-| `z = x -& y`       | `w(z) = max(w(x), w(y)) + 1`   |
-| `z = x & y`        | `w(z) = max(w(x), w(y))`    |
-| `z = Mux(c, x, y)` | `w(z) = max(w(x), w(y))`    |
-| `z = w * y`        | `w(z) = w(x) + w(y)`        |
-| `z = x << n`       | `w(z) = w(x) + maxNum(n)` |
-| `z = x >> n`       | `w(z) = w(x) - minNum(n)` |
-| `z = Cat(x, y)`    | `w(z) = w(x) + w(y)`      |
-| `z = Fill(n, x)`   | `w(z) = w(x) * maxNum(n)` |
-
->where for instance `w(z)` is the bit width of wire `z`, and the `&`
-rule applies to all bitwise logical operations.
+| operation                     | bit width                    |
+| ---------                     | ---------                    |
+| `z = x + y` *or* `z = x +% y` | `w(z) = max(w(x), w(y))`     |
+| `z = x +& y`                  | `w(z) = max(w(x), w(y)) + 1` |
+| `z = x - y` *or* `z = x -% y` | `w(z) = max(w(x), w(y))`     |
+| `z = x -& y`                  | `w(z) = max(w(x), w(y)) + 1` |
+| `z = x & y`                   | `w(z) = max(w(x), w(y))`     |
+| `z = Mux(c, x, y)`            | `w(z) = max(w(x), w(y))`     |
+| `z = w * y`                   | `w(z) = w(x) + w(y)`         |
+| `z = x << n`                  | `w(z) = w(x) + maxNum(n)`    |
+| `z = x >> n`                  | `w(z) = w(x) - minNum(n)`    |
+| `z = Cat(x, y)`               | `w(z) = w(x) + w(y)`         |
+| `z = Fill(n, x)`              | `w(z) = w(x) * n`            |
+
+>where for instance `w(z)` is the bit width of wire `z`, `maxNum(n)` is the maximum number representable
```

Comment:
Can we define `minNum` while visiting?

Reply:
`n` for these shifts must be unsigned, is that right? Curious if `minNum(n)` is ever non-zero.

If `n` is 5 is `maxNum(n)` 5 or is it 7 (largest 3-bit number, where 3 is width of `n`)?

Reply:
> If n is 5 is maxNum(n) 5 or is it 7 (largest 3-bit number, where 3 is width of n)?
Is there a way to check this? Cause I think when using `Fill(6, 1.U)`, I saw a width of `z` to be 7.

Reply:
Per @dtzSiFive comment I think we should use a different thing for the last number in Fill, as there it's an Int n not a UInt-that-has-a-width n